### ISSUE TITLE
Preview Message Thread

### ISSUE THREAD

**Issue by Mae682** [stephancasas/alfred-mouseless-messenger#12]
I'm trying to preview message threads and Shift doesn't do anything. When I export to PDF it is a blank document with the time stamp at the top and the file location at the bottom.

**Comment by NeighNeighNeigh** [comment]
Same, when I quicklook the message it just shows a generic icon

<img width="687" alt="image" src="https://github.com/stephancasas/alfred-mouseless-messenger/assets/4795315/53b32dde-1c12-4678-8018-ab47f951e511">

**Comment by taylorsegell** [comment]
Likewise, did yall happen to find a fix to this?

**Comment by stephancasas** [comment]
The suspect was an issue with the way that JXA handles opaque `id` types on Apple Silicon, but nothing concrete has come up with diagnosing on systems other than my own.

If you'd have the time and would be willing to screen share, I'd really like to get this knocked-out. Thus far, none of the test scripts I've run have yielded any useful information.

**Comment by NeighNeighNeigh** [comment]
I’d be happy to jump on a call with you to help troubleshoot

**Comment by jhurdt** [comment]
@stephancasas I don't believe this issue is limited to Apple Silicon, as I see the same with an Intel i7 Mac. Also, exporting only seems to generate a blank PDF with the default page head and footer (date & time, file url, page number).

**Comment by stephancasas** [comment]
@jhurdt With the help of @NeighNeighNeigh, I was able to debug the issue down to a problem with the SQLite query invocation via shell script. Accessing the SQLite APIs directly has produced consistent results. I have a fix, but have not had time to integrate. Will try to hit it this week though.

**Comment by abilashmram** [comment]
Hi @stephancasas any update on this? Great app btw only hampered by this issue for my day to day use
